run-oval-xml-feed-merge.sh

```diff
@@ -4,7 +4,7 @@
 CONFINEMENT=$(grep confinement $SNAP/meta/snap.yaml|sed 's/^.*: //')
 
 if [ $CONFINEMENT == "strict" ]; then
-  snapcraft-preload $SNAP/bin/oval-xml-feed-merge "$@"
+  $SNAP/bin/snapcraft-preload $SNAP/bin/oval-xml-feed-merge "$@"
```

Comment:
Do we need the fix for `strict` confinement too?

Reply:
We don't need it but I changed it for consistency but I should have referenced it. my bad